vmcnet/utils/io.py

```diff
@@ -121,13 +122,15 @@ def save_vmc_state(directory, name, checkpoint_data: CheckpointData):
       checkpoint_data (CheckpointData): data to save
     """
     (epoch, data, params, optimizer_state, key) = checkpoint_data
+    unfrozen_params: Dict[str,Any] = flax.core.unfreeze(params)
```

Comment:
I think we don't need to worry about freezing and unfreezing anymore, the new version of flax we're using doesn't use frozen dicts anymore